server/models/transactions_by_address.go

```diff
@@ -0,0 +1,9 @@
+package models
+
+import "time"
+
+type TransactionsByAddress struct {
```

Comment:
Is this unused now?

Reply:
still used on inserts (grabber writes to both collections - Transactions and TransactionsByAddress)

Reply:
Is it? I can't find a usage. The upsert is using `bson.M`.

Reply:
Yep you're right, I'm using raw bson instead of model.
Removed